### ISSUE TITLE
Failing to test Stripe on local

### ISSUE THREAD

**Issue by nsunami** [libscie/ResearchEquals.com#721]
I'm exporting the issue here from the PR to preserve the knowledge.

Currently, testing Stripe fails on my local, since the module will not get published. (Even though the Stripe payment succeeds.)

It looks like on my end that `stripe_webhook.ts` may not be firing since the module that I'm trying to publish will continue to have the `published` value as `false`. 

I have not yet figured out how `stripe_webhook.ts` is called in the app.

**From the PR:**
------
I've tried using the ResearchEqual's testing secret, but the publishing did not work on my local.

Steps:

1. Run `stripe listen --forward-to localhost:3000/api/stripe_webhook`
2. Add the webhook signing secret to the env files 
3. Click the publish button
4. Enter the payment info on stripe website. The payment succeeds
5. Module not found error in `[suffix].tsx`

In `[suffix].tsx`, we are looking for a module where `published` is true. But, the module that I'm trying to publish still has its `published` value `false`. I am guessing that the `stripe_webook.ts` is not working correctly on my local.

_Originally posted by @nsunami in https://github.com/libscie/ResearchEquals.com/issues/706#issuecomment-1251415669_

**Comment by nsunami** [comment]
I also opened a related issue where we cannot test Stripe on Heroku: https://github.com/libscie/ResearchEquals.com/issues/736

**Comment by nsunami** [comment]
Shoutout to @chartgerink for the detailed notes on this previous PR! https://github.com/libscie/ResearchEquals.com/pull/161

**Comment by nsunami** [comment]
I was able to set up my own Stripe testing environment on local! 🎉 

How I did:
1. I manually added the product on my Stripe account (e.g., "Collaborative collection")
2. Get the API ID (`price_***`)
3. Update the API ID in the seeding script (`seeds.ts`). In my case, I update this line:

 https://github.com/libscie/ResearchEquals.com/blob/2439a2245e2f05e8c1b2b46b40cbf1d996ca2a83/db/seeds.ts#L454

4. Seed the database (`blitz db seed`)
5. Run the Stripe CLI (`stripe listen --forward-to localhost:3000/api/stripe_webhook`)
6. Match the Stripe secrets with my account (`STRIPE_SECRET_KEY`, `STRIPE_PUBLISHABLE_KEY`, and `STRIPE_WEBHOOK_SECRET`). `STRIPE_WEBHOOK_SECRET` is shown in the console when Stripe CLI is running. 
7. Start the app and complete the transaction 💳
